test/e2e/operator_test.go

```diff
@@ -195,12 +197,26 @@ func TestDefaultContour(t *testing.T) {
 
 func TestContourNodePortService(t *testing.T) {
 	testName := "test-nodeport-contour"
+
+	// Configure the nodeports to match the kind config.
+	http := int32(30080)
+	httpPort := operatorv1alpha1.NodePort{
+		Name:       "http",
+		PortNumber: &http,
+	}
+	https := int32(30443)
+	httpsPort := operatorv1alpha1.NodePort{
+		Name:       "https",
+		PortNumber: &https,
```

Comment:
nit: could use the k8s utils `pointer` package https://pkg.go.dev/k8s.io/utils@v0.0.0-20210305010621-2afb4311ab10/pointer#Int32